quickstarts/aws/amazon-timestream/config.yml

```diff
@@ -0,0 +1,31 @@
+slug: amazon-timestream
+description: |-
+  ## What is Amazon Timestream?
+  Amazon Timestream is a fast, scalable, and serverless time-series database service that makes it easier to store and analyze trillions of events per day up to 1,000 times faster. Amazon Timestream automatically scales up or down to adjust capacity and performance, so that you don’t have to manage the underlying infrastructure.
+  ### Get started!
+  Start monitoring Amazon Timestream by connecting Amazon Web Services (AWS) to New Relic! Take a look at our documentation for information on how to set this integration.
```

Comment:
```suggestion
  Start monitoring Amazon Timestream by connecting Amazon Web Services (AWS) to New Relic! Take a look at our documentation for information on how to set up this integration.
```

Reply:
@seemantk, addressed, I have updated the description.